Makefile

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I think Makefile lowers the barrier to entry: `make example`. It's trivial now, but we're missing proper setup for unit test coverage and CI integration, and those add a bit more boilerplate.